change/@fluentui-react-2020-12-25-15-17-03-master.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
```

Comment:
You'll need to delete and re-generate these change files.

Reply:
Done